blob/conformance.go

```diff
@@ -0,0 +1,460 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package blob provides an easy way to interact with Blob objects within
+// a bucket. It utilizes standard io packages to handle reads and writes.
+package blob
```

Comment:
I propose this be moved to a separate package, `blob/driver/drivertest`. A user of the `blob` package will not want to use this functionality.

Reply:
Done.